src/ocamlorg_frontend/css/styles.css

```diff
@@ -218,40 +223,75 @@ summary::marker {
 }
 
 .header-black {
-    background: #171717;
+  background: #171717;
 }
 
 .code-editor {
-    height: calc(100vh - 80px);
+  height: calc(100vh - 80px);
 }
 
 .code-preview {
-    height: calc(100% - 80px);
+  height: calc(100% - 80px);
 }
 
 .output-container {
-    height: calc(100% - 60px);
+  height: calc(100% - 60px);
 }
 
 /* summary & details expandable section */
 
 details summary svg {
   transform: rotate(90deg);
 }
+
 details[open] summary svg {
   transform: rotate(-90deg);
 }
 
-details[open] summary ~ *
-{	animation: ease-opacity-t-b .5s ease}
+details[open] summary~* {
+  animation: ease-opacity-t-b .5s ease
+}
 
 summary {
   cursor: pointer;
 }
+
 details summary svg svg {
   transition: all 0.3s;
 }
 
 summary::-webkit-details-marker {
   display: none;
 }
+
+/* version list */
+.tr-selected:first-child td {
+  background-color: #F449001A;
+}
+
+.tr-selected:first-of-type td:nth-child(2),
+.tr-selected:first-of-type td:nth-child(4) {
+  font-weight: bold;
+}
+
+
+.tr-selected td:first-of-type {
+  border-top-left-radius: 0.5rem;
+  border-bottom-left-radius: 0.5rem;
+}
+
+.tr-selected td:last-of-type {
+  border-top-right-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+}
+
+.tr-selected:first-of-type td .rounded-full {
+  background-color: #F36528;
+  height: 0.75rem;
+  width: 0.75rem;
+  margin: 0;
+}
+
+.tr-selected:last-of-type td div div {
+  display: none;
+}
```

Comment:
this should probably not be here as you can make an if then else in the templates when rendering tailwind styles